airflow/kubernetes/pod_generator.py

```diff
@@ -50,21 +49,15 @@ class PodDefaults(object):
     def __init__(self):
         pass
 
-    XCOM_MOUNT_PATH = '/airflow/xcom'
-    SIDECAR_CONTAINER_NAME = 'airflow-xcom-sidecar'
+    XCOM_MOUNT_PATH = "/airflow/xcom"
```

Comment:
Can you remove the unrelated changes please -- so it will be easier to just check the changes now and in future too